index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="styles.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=<device-width>, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+  <title>Luis's Portfolio</title>
+</head>
+<body>
+  <header>
+    <div id="welcome" href="#">Welcome</div>
```

Comment:
- [ ] I can see you tried to implement the suggested changes 👍🏽. Kindly note that `href` is only applied to anchor` <a>` tag and not a `div` tag. KIndly fix it 👍🏽 

e.g `<a href="#">Welcome </a>`